external/wakaama/examples/shared/Make.defs

```diff
@@ -38,5 +38,9 @@
 CSRCS += commandline.c connection.c platform.c memtrace.c
 CFLAGS+=-I$(TOPDIR)/../external/wakaama/core
 
+ifeq ($(CONFIG_LWM2M_SECURITY),y)
```

Comment:
Thanks for the changes. Does it mean that wakaama uses tinydtls when CONFIG_LWM2M_SECURITY is enabled?